justfile

```diff
@@ -39,7 +42,16 @@ bootstrap *ARGS:
     #!/usr/bin/env bash
     set -euo pipefail
 
-    python -m pip install --upgrade pip uv
+    # Check if virtual environment exists (looking for bin/activate)
+    if [ -d ".venv" ] && [ -f ".venv/bin/activate" ]; then
```

Comment:
Can we look for `VIRTUAL_ENV`  or some other variable that indicates that we are in an activated venv? 

I use virtualenv-wrapper and don't have an .venv folder in my projects.